openreview/conference/invitation.py

```diff
@@ -274,6 +272,102 @@ def __init__(self, conference):
                 process_string = file_content
             )
 
+class WithdrawnSubmissionInvitation(openreview.Invitation):
+
+    def __init__(self, conference, withdrawn_submission_content=None):
+
+        content = invitations.submission.copy()
+        if withdrawn_submission_content:
+            content = withdrawn_submission_content
+
+        if (conference.submission_stage.double_blind and not conference.submission_stage.reveal_authors_on_withdraw):
+            content['authors'] = {
+                'values': ['Anonymous']
+            }
+            content['authorids'] = {
+                'values-regex': '.*'
+            }
+
+        super(WithdrawnSubmissionInvitation, self).__init__(
+            id=conference.get_invitation_id('Withdrawn_Submission'),
```

Comment:
maybe this should be `conference.submission_stage.get_withdrawn_submission_id()`?

Reply:
Agreed. I have made the changes.